server/functions/.gitignore

```diff
@@ -10,3 +10,4 @@ node_modules/
 
 .runtimeconfig.json
 src/services/config/app.ts
+api_key.json
```

Comment:
Any idea what generates this? I've noticed in some of the recent PRs but I never have this file myself.

Reply:
I don't know this is the first time I have noticed it, should we .gitignore

Reply:
yeah that's fine - do you mind sending what it looks like in slack?

Reply:
I don't have a copy of it? Must be from one of the merged branches

Reply:
Ohh I see.